admin/src/scenes/centers/components/modals/ModalExportMail.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from "react";
+import ModalForm from "../../../../components/modals/ModalForm";
+import { IoMdClose } from "react-icons/io";
+import { FaPlus } from "react-icons/fa";
+import validator from "validator";
+
+export default function ModalExportMail({ isOpen, onSubmit, onCancel }) {
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [emails, setEmails] = React.useState([]);
+  const [value, setValue] = React.useState("");
+  const [error, setError] = React.useState(false);
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+    await onSubmit(emails);
+    setIsLoading(false);
+  };
+
+  const onClickCancel = (e) => {
+    e.preventDefault();
+    onCancel();
+  };
+
+  const handleEmail = (e = null) => {
+    if (e) e.preventDefault();
+    if (validator.isEmail(value)) {
+      setEmails([...emails, value]);
```

Comment:
remarque pas critique
```suggestion
      setEmails((prev)=> [...prev, value]);
```